superset-frontend/src/components/dataViewCommon/TableCollection.tsx

```diff
@@ -34,6 +34,7 @@ interface TableCollectionProps {
 }
 
 export const Table = styled.table`
+  background-color: ${({ theme }) => theme.colors.grayscale.light5};
```

Comment:
looks like this was removed with https://github.com/apache/incubator-superset/commit/820fa473a7fc7571036b86a99ad66396d45e4c2f#diff-e3cf6caa34dff868437ff828d41d0495d30a9701c28d3864c9102a47bf842d22L37

Reply:
@kgabryje is this going to affect your use case?

Reply:
It changes the colour of Query Search table - originally it was transparent, not white. Maybe you could override the background colour just for Query Search?

Reply:
sounds good. I think we will soon be replacing that component with a listview component